### DISCUSSION TITLE
Show Remarks

### DISCUSSION THREAD

**Discussion by Lewis2023** [albar965/littlenavmap#1028] [Ideas]
I have the Flightplan on a second screen. If I hoover over the Remark the info is shown. Now I want to type in on my main screen some infos from the remark in the cockpit (frequency, identifier,...etc.).
But the the infos disappears like a tooltip should do.
A command like "open and close remark" would help to stay on the info as long as I want.

**Comment by albar965** [comment]
What if you use the `Edit userpoint` function. Simply Ctrl+Shift+Click on the userpoint and take notes from the edit dialog. Cancel out when done.

**Reply by Lewis2023** [reply]
OK, that is another way to write notes, but the point is that if I move the mouse away from LNM to MFS Cockpit to set something the notes and remarks disappear in LNM.

**Reply by albar965** [reply]
This is also another way to *view* notes. You can also click on the userpoint and all its information shows up in the information window, BTW.

I'll see if the tooltips can be made persistent.

**Reply by albar965** [reply]
Making tooltips persistent does not make sense IMO:
![image](https://github.com/albar965/littlenavmap/assets/15254956/dd70efc5-b220-4a24-bba5-203c716d919d)
